ch04/src/Solution425.java

```diff
@@ -0,0 +1,15 @@
+public class Solution425 {
+    public static void main(String[] args) {
+        final int plateNumbers = (int)(Math.random() * 10000);
+        char[] plateLettersInChar = new char[3];
+
+        for (int i = 0; i < 3; i++) {
+            final char letter = (char)((int)(Math.random() * ((int)'Z' + 1 - (int)'A') + (int)'A'));
+            plateLettersInChar[i] = letter;
+        }
+

```

Comment:
I love this one! Nice array use!

Also, the way you set the lower and upper limits for the random generation is great! Maybe a bit hard to read.